app/utils/get_filtered_records.py

```diff
@@ -128,7 +128,7 @@ def process_record(record_list):
 def get_filtered_records(research_fields=False, filters=None, columns=None,
                          sampling_start_date=None, sampling_end_date=None,
                          publication_start_date=None, publication_end_date=None, prioritize_estimates=True,
-                         prioritize_estimates_mode=None):
+                         prioritize_estimates_mode=None, include_in_srma=False):
```

Comment:
To me, the False default is a bit unintuitive here - a None default would seem to make more sense. At your discretion though since you've been much more involved in API design, fine with either

Reply:
I think True/False is more intuitive than True/None so I am leaving it